alan-turing-institute/rctab-infrastructure
# Fix constants.py error when using autodoc

### Pull Request Description

Creates logic in `constants.py` that allows it to be read by sphinx autodoc if an environment variable is set (can be set in `conf.py` of autodoc.

### Related Issue(s)

- Closes #6

### Proposed Changes

A check for an environment variable that, if present, will mock `config.require`, `config.require_secret`, and `assert_is_file`.

### Checklist

Please make sure to review and check all the following items before submitting the pull request:

- [x] Documentation has been updated to reflect the changes (if applicable).

### Additional Notes (if applicable)

NA

### Reviewers

@Iain-S 

### How to Test

Set an environment variable that matches that in the code and run it. You can inspect the objects to see what comes back.
